src/endless/res/EndlessUsbTool.css

```diff
@@ -648,13 +648,17 @@ select:disabled {
 
 /* Select Storage Page */
 
-#StorageAgreementText {
+#StorageFooter {
```

Comment:
Fixed the stray horizontal line and wrong padding by applying this rule to the new parent container.